.github/workflows/health-metrics-test.yml

```diff
@@ -19,12 +19,28 @@ jobs:
       - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
         with:
           service_account_key: ${{ secrets.GCP_SA_KEY }}
-      - run: cp config/ci.config.json config/project.json
       - run: yarn install
       - run: yarn build
       - name: Run health-metrics/binary-size test
         run: yarn size-report
+  modular-export-size:
+    name: Binary Size For Modular Exports
+    if: github.event_name == 'push' || !(github.event.pull_request.head.repo.fork)
+    runs-on: ubuntu-latest
+    env:
+      METRICS_SERVICE_URL: ${{ secrets.METRICS_SERVICE_URL }}
+      GITHUB_PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+      GITHUB_PULL_REQUEST_BASE_SHA: ${{ github.event.pull_request.base.sha }}
```

Comment:
It's allowed to set the common ones at the workflow level ([doc](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#env)). It's up to you.

Reply:
Nice! Changed.